scripts/test-make-requirements.sh

```diff
@@ -2,6 +2,8 @@
 set -e
 
 make requirements
+# we want to hard ignore any yarn.lock changes
+git checkout -- yarn.lock
```

Comment:
Is this a stop-gap measure to get tests passing while we figure out why yarn.lock is changing? Seems less than ideal to have yarn installing unlocked versions of libraries for tests.

If this is the permanent fix then I think I want to withdraw my approval for this PR.

Reply:
@millerdev this is a stopgap. i'm still pretty perplexed how this is happening to begin with with the frozen-lockfile flag explicitly used...